pom.xml

```diff
@@ -40,8 +40,8 @@
     </scm>
 
     <properties>
-        <revision>0.0.4</revision>
-        <changelist>-SNAPSHOT</changelist>
+        <revision>1</revision>
```

Comment:
Any reason to have a fixed revision. Jenkins is always wanting the latest?

Reply:
It is true that Jenkins will always show the latest in the Update Center, but I want a way to communicate breaking changes to users, so that if they see the major version increment they will realise something's up and (hopefully) read the release notes about the change.

Manually controlling the `major` component seems the easiest way to communicate this.

Unless there's another way I can do this?

Reply:
That is already possible hpi compatible since exist

Reply:
I know of that, but I believe that is only for dealing with breaking changes in the *configuration format*.

If there is a breaking change in *behaviour*, while the configuration format remains the same, I don't think `<hpi.compatibleSinceVersion>` would be the way to convey this?

Reply:
it is the way to notify users via Jenkins, it may have originally been for settings, but it's generally used for any change that users really need to be aware of, not that it's used much these days.

People who manage plugins via docker images would be unlikely to see it though.

Reply:
In that case, for the benefit of the Docker users, should I manually control the major version?

Reply:
It is up to you :) Feel free to ignore my suggestion.

Reply:
> * Update the red text shown in the Update Center in response to <hpi.compatibleSinceVersion>, so that it warns about more than just configuration format changes

Does sound like a good idea to update the meesage.

Reply:
Cool, after the discussion at https://community.jenkins.io/t/indicating-breaking-changes-in-plugins-with-automated-releases/3285/5 I'll go with the new recommendation of fully automated version numbers + `hpi.compatibleSinceVersion` (if ever needed).